### PR TITLE
core(config): unsized-images to default

### PR DESCRIPTION
**Pull request by lemcardenas**
**Summary**

This PR moves the new unsized-images audit from experimental config to default config. Blocked until #11188 is merged.

**Related Issues/PRs**

#11188 #11218

### CODE REVIEW

**Comment by connorjclark** [comment]
image

Unrelated to this audit, but we should center the thumbnail horizontally.

File: lighthouse-core/test/results/sample_v2.json [hunk 1]
```diff
@@ -581,6 +581,53 @@
...
+          },
+          {
+            "key": "node",
+            "itemType": "node",
+            "text": "Failing Elements"
+          }
+        ],
+        "items": [
+          {
+            "url": "http://localhost:10200/dobetterweb/lighthouse-480x318.jpg",
```
Comment (connorjclark) [inline lighthouse-core/test/results/sample_v2.json:611]: did we ever add to the ImageElements artifact in the test `artifacts.json` the extra data added?

doesnt seem like it. would be nice to add.

File: lighthouse-core/config/default-config.js [hunk 2]
```diff
@@ -552,6 +553,7 @@ const defaultConfig = {
         {id: 'no-vulnerable-libraries', weight: 1, group: 'best-practices-trust-safety'},
         // User Experience
         {id: 'password-inputs-can-be-pasted-into', weight: 1, group: 'best-practices-ux'},
+        {id: 'unsized-images', weight: 1, group: 'best-practices-ux'},
```
[the remark this answers is not in the document]
Comment (lemcardenas) [reply lighthouse-core/config/default-config.js:556]: Sounds good! I'll move it over to the performance group. Is there a guideline for how much weight an audit should have? i.e. in the performance category, most metrics are given a weight of `0`

File: lighthouse-core/test/results/sample_v2.json (hunk 1 above)
Comment (lemcardenas) [reply lighthouse-core/test/results/sample_v2.json:611]: yeah I never manually added the extra data, I'll make sure to do so!

**Pull request merged**